mlflow/entities/trace.py

```diff
@@ -68,8 +77,8 @@ def _repr_mimebundle_(self, include=None, exclude=None):
         in Databricks notebooks to display the Trace object in a nicer UI.
         """
         return {
-            "application/databricks.mlflow.trace": self.to_json(),
-            "text/plain": self.__repr__(),
+            "application/databricks.mlflow.trace": self._serialize_for_mimebundle(),
+            "text/plain": repr(self),
```

Comment:
@dbczumar the final representation looks like this:
```
'{"application/databricks.mlflow.trace": "\\"tr-606b592cdcc24b63a72f9eb742251621\\"", "text/plain": "Trace(request_id=tr-606b592cdcc24b63a72f9eb742251621)"}'
```

due to the JSON formatting, the size of the full bundle is 153 bytes, but that should still allow us to support 60k+ traces

Reply:
we can get rid of the `text/plain` entry but i think this will affect normal jupyter notebook users (they won't see a trace was generated)

Reply:
This is just fine! 60k is reasonable :D